charts/astronomer/templates/astro-ui/astro-ui-deployment.yaml

```diff
@@ -38,6 +38,14 @@ spec:
           imagePullPolicy: {{ .Values.images.astroUI.pullPolicy }}
           resources:
 {{ toYaml .Values.astroUI.resources | indent 12 }}
+          {{- if .Values.astroUI.command }}
+          command:
+            {{- toYaml .Values.astroUI.command | nindent 12 }}
+          {{- end }}
+          {{- if .Values.astroUI.args }}
+          args:
+            {{- toYaml .Values.astroUI.args | nindent 12 }}
+          {{- end }}
```

Comment:
I like this whole block here. It seems like this should be standard. Good stuff.